src/applications/edu-benefits/tests/0993/containers/SinglePageForm.unit.spec.jsx

```diff
@@ -0,0 +1,398 @@
+import React from 'react';
+import { expect } from 'chai';
+import SkinDeep from 'skin-deep';
+import sinon from 'sinon';
+
+import { SinglePageForm } from '../../../0993/containers/SinglePageForm';
+import fullSchema0993 from '../../../0993/config/form';
+
+describe('Schemaform <SinglePageForm>', () => {
+  const location = {
+    pathname: '/testing/0'
+  };
+
+  it('should render', () => {
+    const route = {
+      formConfig: fullSchema0993,
+      pageConfig: {
+        pageKey: 'testPage',
+        schema: {},
+        uiSchema: {},
+        errorMessages: {},
+        title: ''
+      },
+      pageList: [
+        {
+          path: 'testing'
+        }
+      ]
+    };
+    const form = {
+      pages: {
+        testPage: {
+          schema: {},
+          uiSchema: {},
+        }
+      },
+      data: {}
+    };
+    const user = {
+      profile: {
+        savedForms: []
+      },
+      login: {
+        currentlyLoggedIn: true
+      }
+    };
+
+    const tree = SkinDeep.shallowRender(
+      <SinglePageForm form={form} route={route} user={user} location={location}/>
+    );
+    expect(tree.everySubTree('SchemaForm')).not.to.be.empty;
+    expect(tree.everySubTree('Connect(DowntimeNotification)').length).to.equal(1);
```

Comment:
The us-forms-system version has been changed here in that the ProgressButtons check here (shown below) has been replaced here with a DowntimeNotification check.
`    expect(tree.everySubTree('ProgressButton').length).to.equal(2);`